lib/stat_tracker.rb

```diff
@@ -410,7 +206,7 @@ def games_by_all_team_ids
     end
   end
 
-  def games_by_team_id(team_id)
+  def games_by_team_id(team_id) #returns game_team_instance?
```

Comment:
I wonder if we could name this something like game_team_by_id or something that makes it obvious it is for game_team class?

Reply:
@allen4397 That makes sense to me too. Anybody can feel free to make that change on their next branch.